backend/src/bcws/bcws.service.ts

```diff
@@ -238,4 +242,56 @@ export class BcwsService {
 
     return personnel;
   }
+
+  /**
+   * Get all roles
+   * @returns
+   */
+  async getAllRoles(): Promise<BcwsRoleEntity[]> {
+    return this.roleRepository.find();
+  }
+  /**
+   * Returns certifications that are not OFA I, II, or III
+   * Used by CHEFS form
+   * @returns {BcwsCertificationEntity[]} List of certifications
+   *
+   */
+  async getCertificates(): Promise<BcwsCertificationEntity[]> {
```

Comment:
Would suggest the OFA stuff as a boolean param; `includeOFA` or something

Reply:
oh yeah good call